File: javascript/easy/RemoveDuplicate.js
const remove = nums => {
  let i = 0
  for (let j = 1; j < nums.length; j++) {
    if (nums[i] !== nums[j]) {
      i++
      nums[i] = nums[j]
    }
  }
  return i + 1
}

console.log(remove([1, 222, 2222, 33333, 33333]))

File: code/python/easy/RemoveDuplicate.py
def remove(nums: list) -> int:
    i = 0
    for j in range(1, len(nums)):
        if nums[i] != nums[j]:
            i += 1
            nums[i] = nums[j]
    return i + 1


if __name__ == '__main__':
    print(remove([1, 2, 2, 2, 2, 3, 4, 5]))
